src/ejabberd_s2s.erl

```diff
@@ -434,19 +434,18 @@ new_connection(MyServer, Server, From, FromTo = {FromServer, ToServer},
     end,
     TRes.
 
-
 -spec max_s2s_connections_number(fromto()) -> pos_integer().
 max_s2s_connections_number({From, To}) ->
-    case acl:match_rule(
-           From, max_s2s_connections, jid:make(<<"">>, To, <<"">>)) of
+    {ok, HostType} = mongoose_domain_api:get_host_type(From),
+    case acl:match_rule(HostType, max_s2s_connections, jid:make(<<"">>, To, <<"">>)) of
```

Comment:
`<<"">>` triggers me

Reply:
Oh, it was a copy-paste... :see_no_evil: